az-digital/az_quickstart
# #2827: Use stable embed release and final entity embed patch for CKEditor5 support.

## Description

- Updates embed module version to latest release (where CKEditor5 support is now included)
- Removes CKEditor5 patch for embed module
- Updates CKEditor5 patch for entity_embed module to use patch for commit that was merged (but not yet included in a release)
- Re-exported all text editor, text format and embed button configs

## Related issues




We thought these changes were needed to fix #2827 but it turns out that issue was caused by composer failures on Pantheon multidevs.

## How to test


Verify embed buttons still work in CKeditor

Probo site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-2841.probo.build/

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
